doc/user_guide/psyclone_command.rst

```diff
@@ -412,3 +410,20 @@ PSyclone will check the kernel output directory and if a transformed
 version of that kernel is already present then that will be
 used. Note, if the kernel file on disk does not match with what would
 be generated then PSyclone will raise an exception.
+
+Enabling the Logging Infrastructure
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+PSyclone supports logging which can provide additional information
+on what is happening inside PSyclone. This logging will also
+control the behaviour of any logging calls inside a user script.
+
+Logging output can be controlled through the `--log-level` option.
```

Comment:
Please use double backticks as in the section above, they look much better in my opinion to refer to flags and options:
![image](https://github.com/user-attachments/assets/6dd7d395-8cd3-4829-baec-562001b36acc)

Also in the doc/developer_guide/coding-style.rst when it makes sense, e.g. referring to imports logging, warning, caplog, ... the logging levels as well.

Reply:
Done